app/src/routes/Games/components/GamesHeader.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "../../Events/components/EventsHeader.scss"
```

Comment:
this import should be removed - if you need to use these styles, they should be extracted to some shared component